client/src/hooks/queries/donors/useProjectsDonors.ts

```diff
@@ -21,16 +24,9 @@ export default function useProjectsDonors(epoch?: number): {
 
   const projectsDonorsResults: UseQueryResult<ProjectDonor[]>[] = useQueries({
     queries: (projectsAddresses || []).map(projectAddress => ({
-      enabled: !!projectsAddresses && isDecisionWindowOpen !== undefined,
-      queryFn: () =>
-        apiGetProjectDonors(
-          projectAddress,
-          epoch || (isDecisionWindowOpen ? currentEpoch! - 1 : currentEpoch!),
-        ),
-      queryKey: QUERY_KEYS.projectDonors(
-        projectAddress,
-        epoch || (isDecisionWindowOpen ? currentEpoch! - 1 : currentEpoch!),
-      ),
+      enabled: !!projectsAddresses && isEnabled(isDecisionWindowOpen, epoch),
```

Comment:
I would create variable `isEnabled` in hook scope instead of method outside the hook.

Reply:
But it's exported and used in `client/src/hooks/queries/useProjectsIpfsWithRewards.ts`.

Reply:
Done.